cmd/swapd/main.go

```diff
@@ -681,12 +686,12 @@ func getProtocolInstances(c *cli.Context, cfg common.Config,
 		TransferBack: c.Bool(flagTransferBack),
 	}
 
-	xmrtaker, err := xmrtaker.NewInstance(xmrtakerCfg)
+	xmrTaker, err := xmrtaker.NewInstance(xmrtakerCfg)
```

Comment:
`xmrtaker` and `xmrmaker` variable changed to mixed case to not conflict with the imported package name.